projboard/tests/test_client.py

```diff
@@ -1,7 +1,39 @@
 import pytest
+from projboard.models.article import Article
 
 
 @pytest.mark.django_db
 def test_homepage(client):
     response = client.get("/")
     assert response.status_code == 200
+
+
+@pytest.mark.django_db
+def test_create_article_get(client):
+    response = client.get('/create_article/')
+    assert response.status_code == 200
+
+
+@pytest.mark.django_db
+def test_create_article_post(client, article, django_capture_on_commit_callbacks):
+    assert len(Article.search_by_title(article.title + ' form')) == 0
+
+    with django_capture_on_commit_callbacks(execute=True):
+        response = client.post(
+            '/create_article/', {
+                'title': article.title + ' form',
+                'subject_id': 1,
+                'user_id': 1,
+                'date': article.date,
+                'content': article.content
+            })
+
+    assert response.status_code == 200
+
+    # check if sent properly
+    user = response.context['articles'][0].user_id
+    for ar in response.context['articles']:
+        assert ar in Article.search_by_user(user)
+
+    # check if the new article is created
+    assert len(Article.search_by_title(article.title + ' form')) == 1
```

Comment:
Please avoid call to db, it is client UI
Consider view returns something after querying the db and return
value you can assert